worker-sys/src/types/durable_object/sql_storage.rs

```diff
@@ -0,0 +1,24 @@
+use js_sys::JsString;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+extern "C" {
+    #[wasm_bindgen(extends=js_sys::Object)]
+    pub type DurableObjectSqlStorage;
+
+    #[wasm_bindgen(method, catch, js_name=exec, variadic)]
+    pub fn exec(
+        this: &DurableObjectSqlStorage,
+        query: JsString,
+        args: Vec<JsValue>,
+    ) -> Result<DurableObjectSqlStorageCursor, JsValue>;
+}
+
+#[wasm_bindgen]
+extern "C" {
```

Comment:
Few more methods that we should implement: https://workers-types.pages.dev/#SqlStorageCursor